### DISCUSSION TITLE
Move 'next in sequence' from System > Defaults to System > Sequences

### DISCUSSION THREAD

**Discussion by ehuelsmann** [ledgersmb/LedgerSMB#5816] [Design]
All the sequences named in the "Next in Sequence" section are also named in the System > Sequences page. It'd make more sense to move these sequences entirely out of the "Defaults" page, now that there's a separate page anyway.

**Comment by einhverfr** [comment]
It isn't just a separate page.  It is a separate db table and much more.  I think this needs design but generally agree we would do well to move all sequences to that interface.
